compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         self.max_workers_per_node = 1
         if executor is None:
             executor = HighThroughputExecutor(  # type: ignore
-                *args, address=address, **kwargs
+                *args, address=address, heartbeat_period=heartbeat_period, **kwargs
```

Comment:
At the point we're no longer including the `_s` suffix (originally intended as code conveyance of the unit), then it's the exact same as Parsl's ... I recognize that we use it in a couple of other spots (`ReportingThread`, parent class), but I'm wondering if there's a way to just remove it entirely now.

Reply:
We could grab it via `kwargs.get("heartbeat_period", 30)`, but I'm not sure what that gets us.

Reply:
No ... I'm hinting a possible refactor of this area.  Likely a smidgen bigger than you are intending this with this "easy peezy" PR.

Reply:
Ok, got it. Let's hold off for now then and discuss out-of-band.